src/dash-table/components/Table/props.ts

```diff
@@ -234,6 +230,7 @@ export interface IUSerInterfaceTooltip {
 }
 
 export interface IState {
+    activeMenu?: 'show/hide';
```

Comment:
The idea I guess is we may later add other menus so this would become an enum?

Reply:
Yes. This is all highly experimental and probably won't survive the coming round of UI rework for the column ops.. but, yes.